lib/features/auth/register/ui/register_view.dart

```diff
@@ -0,0 +1,154 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_modular/flutter_modular.dart'
+    hide ModularWatchExtension;
+import 'package:live_score/core/constants/routes.dart';
+import 'package:live_score/core/exceptions/auth_error.dart';
+import 'package:live_score/core/extensions/localization/app_localizations_context.dart';
+import 'package:live_score/core/theme/app_theme.dart';
+import 'package:live_score/core/ui/confirm_button.dart';
+import 'package:live_score/core/ui/form_text_field.dart';
+import 'package:live_score/features/auth/register/cubit/register_cubit.dart';
+
+class RegisterView extends StatelessWidget {
+  const RegisterView({super.key});
+  RegisterCubit _cubit(BuildContext context) => context.read<RegisterCubit>();
+
+  @override
+  Widget build(BuildContext context) {
+    final usernameController = TextEditingController();
+    final emailController = TextEditingController();
+    final passwordController = TextEditingController();
+    final confirmPasswordController = TextEditingController();
+
+    return Scaffold(
+      backgroundColor: AppTheme.primary,
+      appBar: AppBar(
+        automaticallyImplyLeading: true,
+        iconTheme: const IconThemeData(
+          color: AppTheme.onSecondary,
+        ),
+        backgroundColor: AppTheme.transparent,
+        elevation: 0,
+        title: Text(
+          context.localizations.registerTitle,
+          style: const TextStyle(
+            color: AppTheme.onSecondary,
+            fontWeight: FontWeight.bold,
+          ),
+        ),
+        toolbarHeight: 100,
+      ),
+      body: BlocConsumer<RegisterCubit, RegisterState>(
+        listener: (context, state) {
```

Comment:
Extract listener to separated method

`void onRegisterStateChanged(BuildContext context, RegisterState registerState)`